js/form.js

```diff
@@ -0,0 +1,67 @@
+const email = document.getElementById('email');
+const error = document.querySelector('.error');
+const form = document.querySelector('form');
+let errorState = 'valid';
+
+function addEvent(element, event, callback) {
+  const previousEventCallBack = element[`on${event}`];
+  element[`on${event}`] = function (e) {
+    let output = callback(e);
+
+    // A callback that returns `false` stops the callback chain
+    // and interrupts the execution of the event callback.
```

Comment:
- [ ] [OPTIONAL] Please, consider removing the comments from your code, to make it more clear. Your code have to talk about it by itself 👍